modules/refreshing-service-ids-ibm-cloud.adoc

```diff
@@ -4,29 +4,30 @@
 
 :_content-type: PROCEDURE
 [id="refreshing-service-ids-ibm-cloud_{context}"]
-= Rotating API keys for IBM Cloud
+= Rotating API keys
```

Comment:
I don't think we can make this doc so generic by removing IBM Cloud in it because this command is valid only for IBM Cloud and PowerVS providers, prefer having a different page for the powervs provider.

Reply:
cc @jeffnowicki

Reply:
@mkumatag , request to please take a look at this comment https://github.com/openshift/openshift-docs/pull/47558#discussion_r921561019 where , we had a request to utilize the same module.

Please let me know your thoughts on above. Thanks !

Reply:
@mkumatag The module is included **only** in the IBM Cloud and Power VS assemblies. Therefore, its clear to the reader even without the mention of IBM Cloud or Power VS. @alishaIBM needs to follow the [OpenShift Peer Review Checklist > (for writers)](https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.bp4j7064auhh) otherwise her PR won't get approved by the peer-review squad.

Reply:
Then LGTM.